networking_ccloud/tools/netbox_config_gen.py

```diff
@@ -253,20 +266,7 @@ def get_connected_devices(self, switches: List[NbRecord]) -> Tuple[Set[NbRecord]
                           f"device {far_device.name} port {iface.connected_endpoint.name} "
                           f"role {far_device.device_role.name}")
 
-                if iface.lag is not None:
-                    # We fill this list only if this filter comes back False, so this is assumed to be unique
-                    lags = filter(lambda x: x.switch == switch.name and x.name == iface.lag.name and x.lacp,
-                                  device_ports_map.get(far_device.name, []))
-                    lag: Optional[conf.SwitchPort] = next(lags, None)
-                    if lag:
-                        lag.members.append(iface.name)
-                    else:
-                        device_ports_map[far_device].append(conf.SwitchPort(switch=switch.name, lacp=True,
-                                                                            name=iface.lag.name, members=[iface.name]))
-                else:
-                    # not a lag
-                    device_ports_map[far_device].append(conf.SwitchPort(switch=switch.name, name=iface.name))
-        hgs = [conf.Hostgroup(binding_hosts=[h.name], direct_binding=True, members=m)
+        hgs = [conf.Hostgroup(binding_hosts=[h.name], direct_binding=True, members=self.sort_switchports(m))
```

Comment:
As you sort the switchports, should we sort Hostgroups as well? Sorting criteria would be a bit harded, but I'd go either with physnet name (this was what we went with on ACI, but we would need to find out the name by crossreferencing it; could be done once the whole config is assembled) or we sort it by the alphabetically first binding host of each group.

Reply:
I made a fixup commit for this. I create the HGs individually. Metagroup-HGs have anyhow just a single binding host. They will be ordered by `binding_hosts[0]` and all their direct binding HGs are before them in alphabetical order. We also have interconnect HGs which also only have exactly one binding host. Then, and that is just temporary (I assume), we have some leftover direct binding hosts (asrs, apods) that are direct binding and thus also only have one binding host. So, yes, I'd like to use the binding host option